static/src/js/zustand/slices/__tests__/createCollectionSlice.test.ts

```diff
@@ -710,10 +728,28 @@ describe('createCollectionSlice', () => {
           expect(granules.getGranules).toHaveBeenNthCalledWith(1)
           expect(granules.getGranules).toHaveBeenNthCalledWith(2)
 
-          expect(actions.getColorMap).toHaveBeenCalledTimes(1)
-          expect(actions.getColorMap).toHaveBeenCalledWith({
+          // Verify getColorMap is called for each GIBS tag
+          expect(actions.getColorMap).toHaveBeenCalledTimes(3)
+          expect(actions.getColorMap).toHaveBeenNthCalledWith(1, {
             product: 'AIRS_Prata_SO2_Index_Day'
           })
+
+          expect(setMapLayers).toHaveBeenCalledTimes(1)
+          expect(setMapLayers).toHaveBeenCalledWith(
+            'C10000000000-EDSC',
+            [{ product: 'AIRS_Prata_SO2_Index_Day' },
+              { product: 'MODIS_Terra_Aerosol' },
+              { product: 'VIIRS_SNPP_CorrectedReflectance_TrueColor' }
+            ]
+          )
+
+          expect(actions.getColorMap).toHaveBeenNthCalledWith(2, {
+            product: 'MODIS_Terra_Aerosol'
+          })
+
+          expect(actions.getColorMap).toHaveBeenNthCalledWith(3, {
+            product: 'VIIRS_SNPP_CorrectedReflectance_TrueColor'
+          })
```

Comment:
You have 
```
expect(actions.getColorMap).toHaveBeenCalledTimes(3)
expect(actions.getColorMap).toHaveBeenNthCalledWith(1, {
expect(setMapLayers).toHaveBeenCalledTimes(1)
expect(setMapLayers).toHaveBeenCalledWith(
expect(actions.getColorMap).toHaveBeenNthCalledWith(2, {
expect(actions.getColorMap).toHaveBeenNthCalledWith(3, {
```

Keep assertions on the same data next to each to keep it easier to read